spec/cypress/support/commands.js

```diff
@@ -47,3 +47,9 @@ Cypress.Commands.add("clickBackLink", () => {
 Cypress.Commands.add("clickCommitButton", () => {
   cy.get("[name=commit]").click();
 });
+
+Cypress.Commands.add("chooseNameAndEmailForUser", (name, email) => {
+  cy.get("input[name*=full_name").type(name);
```

Comment:
I'd be tempted to just go with `*=name` here! Not that it makes much difference

Reply:
I'd be worried that would catch names for other things, like organisations